PowerEditor/src/DarkMode/IatHook.h

```diff
@@ -0,0 +1,94 @@
+// This file contains code from
```

Comment:
Notepad++ has GPL3 license. See other files for example.

Reply:
I believe MIT would be compatible, since scintilla's license is very similar to MIT and BSD